roles/bash-for-pressbooks-dev/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+- name: Add hh PPA
+  apt_repository:
+    repo: "ppa:ultradvorka/ppa"
+    update_cache: yes
+- name: Install software referenced in .bash_aliases
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
```

Comment:
You can put any apt packages you need (that don't require their own PPA) in this list of `apt_packages_custom`: https://github.com/pressbooks/trellis/blob/master/group_vars/all/main.yml#L5